FamilySearchIosSampleApp/MemoriesVC.swift

```diff
@@ -98,7 +98,7 @@ class MemoriesVC : UICollectionViewController
         let cell = collectionView.dequeueReusableCellWithReuseIdentifier("MemoryCell", forIndexPath: indexPath) as! MemoryCell
         
         let linkHref = arrayOfImageThumbnailHrefs.objectAtIndex(indexPath.row) as! String
-        Utilities.getImageFromUrl(linkHref, accessToken: accessToken) { (data, response, error)  in
+        Utilities.getImageFromUrl(linkHref, accessToken: accessToken!) { (data, response, error)  in
             dispatch_async(dispatch_get_main_queue()) { () -> Void in
                 cell.memoryImageView.image = UIImage(data: data!)
```

Comment:
Force unwrap of `data` isn't safe since it could be nil for lots of different reasons.